### ISSUE TITLE
Instructions for checking out/testing development branches

### ISSUE THREAD

**Issue by almcewen0** [nanograv/PINT#1382]
Per some discussion on the PINT call, checking out branches in PRs is behind a permission wall for some users (i.e. me). This makes checking out/testing branches (especially those with new features) a little more opaque. In the spirit of getting more involved in development, it would be nice to have this process described clearly somewhere in the documentation. 

Some specific questions to be addressed:
1. when a PR is posted, what is the best way to pull down the updated code and test it? as noted on the call, i do not have an option to do this easily on GH (David/Scott mentioned that for them, there is a button at the bottom of PR pages to quickly checkout a branch). For reference, here is a screenshot of the area stating permission requirements where those options were available to David:
![Screen Shot 2022-08-18 at 10 17 57](https://user-images.githubusercontent.com/39491211/185433240-e01f834c-da0b-4da8-a87c-7540fe527bd8.png)

2. what is the standard procedure for doing checks/validating PRs? this is somewhat subjective, but it would be good to have some clear outline that would facilitate more active development from less experienced members.

**Comment by aarchiba** [comment]
I wasn't aware of any particular support from github, but PRs are just branches in the author's Github repo, so I just do something like:

```
git remote add [author's username] [author's repo URL]
git fetch --all
git checkout [author's username]/[PR's name]
```

**Comment by abhisrkckl** [comment]
Fixed in #1447
